src/bin/ncc.c

```diff
@@ -25,11 +25,6 @@
 #include "preprocess.h"
 #include "util.h"
 
-int __lsan_is_turned_off()
-{
-	return 1;
-}
```

Comment:
This can be added back now - I'm guessing you removed it because it was tripping `-Wstrict-prototypes`, but I've fixed it now.